runtime-light/crypto/crypto-builtins.cpp

```diff
@@ -0,0 +1,28 @@
+//  Compiler for PHP (aka KPHP)
+//  Copyright (c) 2024 LLC «V Kontakte»
+//  Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#include "runtime-light/crypto/crypto-builtins.h"
+
+#include "runtime-light/tl/tl-core.h"
+#include "runtime-light/streams/interface.h"
+
+// Crypto-Specific TL magics
+constexpr uint32_t TL_GET_CRYPTOSECURE_PSEUDORANDOM_BYTES = 0x2491'b81d;
+
+task_t<Optional<string>> f$openssl_random_pseudo_bytes(int64_t length) {
+  if (length <= 0 || length > string::max_size()) {
+    co_return false;
+  }
+
+  tl::TLBuffer buffer;
+  buffer.store_trivial<uint32_t>(TL_GET_CRYPTOSECURE_PSEUDORANDOM_BYTES);
+  buffer.store_trivial<int32_t>(length);
```

Comment:
From `openssl_random_pseudo_bytes` docs: `The length of the desired string of bytes. Must be a positive integer less than or equal to 2147483647`. I think we need to check `length` here as we do in `rpc`.

Reply:
`string::max_size() < 2147483647`. In old runtime we had the same restrictions, basically we cannot satisfy the docs fully because of current `string` implementation